scripts/install

```diff
@@ -15,14 +15,13 @@ source /usr/share/yunohost/helpers
 ynh_script_progression --message="Setting up source files..." --weight=4
 
 # Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source --dest_dir=$install_dir --source_id=$YNH_ARCH
+ynh_setup_source --dest_dir=$install_dir
 
 pushd "$install_dir"
 	gzip --decompress $YNH_ARCH.gz
 	mv $YNH_ARCH $app
```

Comment:
```suggestion
	gzip --decompress main.gz
	mv main $app
```

With the new sources thing, it's probably gonna be named "main" (same thing during upgrade)